exchanges/lakebtc/lakebtc_websocket.go

```diff
@@ -154,8 +155,14 @@ func (l *LakeBTC) wsHandleIncomingData() {
 				l.Websocket.DataHandler <- err
 			}
 		}
+
+		wsRunningURL, err := l.API.Endpoints.GetURL(exchange.WebsocketSpot)
+		if err != nil {
+			return
+		}
```

Comment:
Can you please move this outside of the for scope because this is not needed to be done every single cycle.

Reply:
Yes of course